python/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+find_package(PythonInterp REQUIRED)
+find_package(PythonLibs ${PYTHON_VERSION_STRING} EXACT REQUIRED)
+include_directories(${PYTHON_INCLUDE_DIRS})
+
+add_library(kenlm_python MODULE kenlm.cpp score_sentence.cc)
+set_target_properties(kenlm_python PROPERTIES OUTPUT_NAME kenlm)
+set_target_properties(kenlm_python PROPERTIES PREFIX "")
+
+if(APPLE)
+  set_target_properties(kenlm_python PROPERTIES SUFFIX ".so")
```

Comment:
I thought apple was dyld not .so?

Reply:
that is indeed the default extension for shared libs, but python wants ".so" on all unix